src/core/ext/filters/client_channel/resolver/dns/c_ares/dns_resolver_ares.cc

```diff
@@ -433,8 +434,9 @@ static grpc_address_resolver_vtable ares_resolver = {
     grpc_resolve_address_ares, blocking_resolve_address_ares};
 
 static bool should_use_ares(const char* resolver_env) {
-  return resolver_env == nullptr || strlen(resolver_env) == 0 ||
-         gpr_stricmp(resolver_env, "ares") == 0;
+  return !g_custom_iomgr_enabled &&
```

Comment:
Can we add a `TODO` comment that this can `g_custom_iomgr_enabled` check can go away after supporting c-ares on gevent (since the only other custom iomgr user, libuv, compiles out c-ares).

Reply:
`TODO` comment added:

> // TODO(lidiz): Remove the "g_custom_iomgr_enabled" flag once c-ares support
>  // custom IO managers (e.g. gevent).